feature/experimental/bgp/otg_tests/bgp_tcp_mss_path_mtu/bgp_tcp_mss_path_mtu_test.go

```diff
@@ -0,0 +1,435 @@
+		nv.Enabled = ygot.Bool(true)
+		if authPwd != "" {
+			nv.AuthPassword = ygot.String(authPWd)
+		}
+		if nbr.isV4 {
+			af4 := nv.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST)
+			af4.Enabled = ygot.Bool(true)
+		} else {
+			af6 := nv.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST)
+			af6.Enabled = ygot.Bool(true)
+		}
+	}
+	return niProto
+}
+
+// verifyBGPTelemetry checks that the dut has an established BGP session with reasonable settings.
+func verifyBGPTelemetry(t *testing.T, dut *ondatra.DUTDevice, nbrIP []string) {
+	statePath := gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP").Bgp()
+	for _, nbr := range nbrIP {
+		nbrPath := statePath.Neighbor(nbr)
+		t.Logf("Waiting for BGP neighbor to establish...")
+		status, ok := gnmi.Watch(t, dut, nbrPath.SessionState().State(), time.Minute, func(val *ygnmi.Value[oc.E_Bgp_Neighbor_SessionState]) bool {
+			state, ok := val.Val()
+			return ok && state == oc.Bgp_Neighbor_SessionState_ESTABLISHED
+		}).Await(t)
+		if !ok {
+			fptest.LogQuery(t, "BGP reported state", nbrPath.State(), gnmi.Get(t, dut, nbrPath.State()))
+			t.Fatal("No BGP neighbor formed")
+		}
+		state, _ := status.Val()
+		if want := oc.Bgp_Neighbor_SessionState_ESTABLISHED; state != want {
+			t.Errorf("BGP peer %s status got %d, want %d", nbr, state, want)
+		}
+		t.Logf("BGP adjacency for %s: %s", nbr, state)
+	}
+}
+
+func configureISIS(t *testing.T, dut *ondatra.DUTDevice, intfName []string, dutAreaAddress, dutSysID string) {
+	d := &oc.Root{}
+	dutConfIsisPath := gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_ISIS, isisInstance)
+	netInstance := d.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(dut))
+	prot := netInstance.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_ISIS, isisInstance)
+	prot.Enabled = ygot.Bool(true)
+	isis := prot.GetOrCreateIsis()
+	globalISIS := isis.GetOrCreateGlobal()
+	globalISIS.LevelCapability = oc.Isis_LevelType_LEVEL_2
+	globalISIS.Net = []string{fmt.Sprintf("%v.%v.00", dutAreaAddress, dutSysID)}
+	globalISIS.GetOrCreateAf(oc.IsisTypes_AFI_TYPE_IPV4, oc.IsisTypes_SAFI_TYPE_UNICAST).Enabled = ygot.Bool(true)
+	isisLevel2 := isis.GetOrCreateLevel(2)
+	isisLevel2.MetricStyle = oc.Isis_MetricStyle_WIDE_METRIC
+
+	for _, intf := range intfName {
+		isisIntf := isis.GetOrCreateInterface(intf)
+		isisIntf.Enabled = ygot.Bool(true)
+		isisIntf.CircuitType = oc.Isis_CircuitType_POINT_TO_POINT
+		isisIntfLevel := isisIntf.GetOrCreateLevel(2)
+		isisIntfLevel.Enabled = ygot.Bool(true)
+		isisIntfLevelAfi := isisIntfLevel.GetOrCreateAf(oc.IsisTypes_AFI_TYPE_IPV4, oc.IsisTypes_SAFI_TYPE_UNICAST)
+		isisIntfLevelAfi.Metric = ygot.Uint32(200)
+		isisIntfLevelAfi.Enabled = ygot.Bool(true)
+	}
+	gnmi.Replace(t, dut, dutConfIsisPath.Config(), prot)
+}
+
+func configureOTG(t *testing.T, otg *otg.OTG) gosnappi.Config {
+
```

Comment:
no blank line for consistency with other functions

Reply:
updated